packages/ics721/src/execute.rs

```diff
@@ -63,9 +82,6 @@ where
                 token_id,
                 msg,
             }) => self.execute_receive_nft(deps, env, info, token_id, sender, msg),
-            ExecuteMsg::ReceiveProxyNft { eyeball, msg } => {
-                self.execute_receive_proxy_nft(deps, env, info, eyeball, msg)
-            }
```

Comment:
`ReceiveProxyNft` has been removed, instead `ReceiveNft` handles both: msgs from collection and outgoing proxy contract